pkg/operator/ceph/cluster/osd/osd.go

```diff
@@ -239,8 +240,10 @@ func (c *Cluster) startProvisioningOverPVCs(config *provisionConfig) {
 		logger.Debugf("osdProps are %+v", osdProps)
 
 		// If the deviceSet template has "encrypted" but the Ceph version is not compatible
-		if osdProps.encrypted && !c.clusterInfo.CephVersion.IsAtLeast(cephVolumeRawEncryptionModeMinCephVersion) {
-			errMsg := fmt.Sprintf("failed to validate storageClassDeviceSet %q. min required ceph version to support encryption is %q", volume.Name, cephVolumeRawEncryptionModeMinCephVersion.String())
+		if osdProps.encrypted &&
+			!c.clusterInfo.CephVersion.IsAtLeast(cephVolumeRawEncryptionModeMinNautilusCephVersion) &&
+			!c.clusterInfo.CephVersion.IsAtLeast(cephVolumeRawEncryptionModeMinOctopusCephVersion) {
```

Comment:
This doesn't look like it will allow for any nautilus version. The octopus check will override the nautilus check.